src/App.css

```diff
@@ -53,92 +61,217 @@ li {
+  font-size: 1rem;
+  text-decoration: none;
+  color: #121212;
+  opacity: 1;
+  font-weight: 600;
+  border-right: 0.2rem solid #535353;
   padding-right: 0.7rem;
-  color: #007c95;
+  transition: all 0.5s ease;
 }
 
 .book-container {
+  margin: 2rem 0;
+  border: 0.2rem solid #e8e8e8;
   padding: 2rem;
   display: flex;
   gap: 2rem;
+  align-items: center;
 }
 
 .book {
-  width: 50%;
+  width: 40%;
 }
 
 .category {
-  font-size: 1.5rem;
-  color: #535353;
-  font-weight: 500;
+  font-size: 1rem;
+  font-weight: 700;
+  color: #121212;
+  opacity: 0.5;
 }
 
 .bookname {
-  font-size: 2rem;
-  font-weight: 600;
-  color: #000;
+  font-size: 1.5rem;
+  font-weight: 700;
+  color: #121212;
 }
 
 .authorname {
   font-size: 1.5rem;
-  color: #007c95;
-  font-weight: 500;
+  font-family: 'Roboto Slab', serif;
+  color: #4386bf;
+  font-weight: 300;
+}
+
+.acctionbuttons {
+  display: flex;
+  gap: 1rem;
+  margin-top: 1rem;
+}
+
+.acctionbuttons button {
+  padding: 0;
+  font-size: 1rem;
+  font-family: 'Roboto Slab', serif;
+  font-weight: 300;
+  color: #4386bf;
+  background-color: transparent;
+  border: none;
+  border-right: 0.1rem solid #4386bf;
+  padding-right: 0.5rem;
+  transition: all 0.5s ease;
+}
+
+.acctionbuttons button:hover {
+  cursor: pointer;
+  background-color: #4386bf;
+  color: #fff;
+  padding: 0.5rem;
 }
 
 .bookstatus {
-  width: 20%;
+  width: 30%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.img-progress {
+  width: 50%;
+}
+
+.progress-procentage {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
 }
 
 .bookstatus h2 {
-  font-size: 1.5rem;
+  margin: 0;
+  padding: 0;
+  width: 100%;
+  font-size: 2rem;
+  text-align: center;
+  color: #121212;
+  font-weight: 300;
 }
 
-.fa-spinner {
+.bookstatus p {
+  margin: 0;
+  padding: 0;
+  width: 100%;
   font-size: 2rem;
-  font-weight: 600;
-  color: #007c95;
+  text-align: center;
+  color: #121212;
+  opacity: 0.5;
+  font-weight: 300;
 }
 
 .bookchapter {
   width: 30%;
+  display: flex;
+  flex-direction: column;
+  gap: 1.2rem;
 }
 
 .bookchapter h2 {
-  font-size: 1.8rem;
+  font-size: 1rem;
+  color: #121212;
+  opacity: 0.5;
+  font-family: 'Roboto Slab', serif;
+  font-weight: 400;
 }
 
 .bookchapter h3 {
-  font-size: 1.5rem;
+  font-size: 1rem;
+  color: #121212;
+  font-family: 'Roboto Slab', serif;
+  font-weight: 400;
 }
 
-.form {
-  padding: 2rem;
+.bookchapter button {
+  font-size: 1rem;
+  font-family: 'Roboto Slab', serif;
+  font-weight: 300;
+  color: #fff;
+  background-color: #0290ff;
+  border: none;
+  padding: 0.8rem 1rem;
+  border-radius: 0.3rem;
 }
 
 .form h2 {
-  color: #535353;
+  font-size: 1.5rem;
+  font-weight: 700;
+  color: #585555;
+}
+
+.formcontainer {
+  margin-top: 1rem;
+  display: flex;
+  gap: 1rem;
+}
+
+.book-name {
+  width: 35%;
+  border: 0.1rem solid #535353;
+}
+
+.book-name::placeholder {
+  font-size: 1rem;
+  font-family: 'Montserrat', sans-serif;
```

Comment:
- [ ] It would be better if we implement CSS DRY principle. `Font-family: 'Montserrat', sans-serif;` has been repeated so has some of the rules. Kindly check the screenshot below to fix this issue
![Screenshot from 2022-07-28 18-45-30](https://user-images.githubusercontent.com/46388113/181614506-6f7d5e6d-8225-46ef-8cd9-945f659d0c2b.png)